### PR TITLE
photoqt: new port in graphics

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

@reneeotten @pmetzger I think this is fixed now. Sorry for a delay.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 14.1.1
Xcode 15

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
@reneeotten @pmetzger @herbygillot This should be mergeable now.

**Review by reneeotten — CHANGES_REQUESTED** [review]
please add "github.tarball_from"

**Comment by barracuda156** [comment]
> please add "github.tarball_from"

@reneeotten How to use it with two supports correctly?

File: graphics/photoqt/Portfile [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    port:vips \
+                    port:webp
+
+    # We have to use ImageMagick here: https://github.com/luspi/photoqt/issues/19
+    configure.args-append \
+                    -DCHROMECAST=OFF \
+                    -DDEVIL=OFF \
+                    -DFREEIMAGE=OFF \
+                    -DGRAPHICSMAGICK=OFF \
+                    -DIMAGEMAGICK=OFF \
```
Comment (reneeotten) [inline graphics/photoqt/Portfile:51]: based on the comment above, shouldn't this be set to ON?

**Comment by reneeotten** [comment]
> > please add "github.tarball_from"
> 
> @reneeotten How to use it with two subports correctly?

set it in both subports?

File: graphics/photoqt/Portfile (hunk 1 above)
Comment (barracuda156) [reply graphics/photoqt/Portfile:51]: Indeed. Looks like I lost it in rebasing. Will fix now.

**Comment by barracuda156** [comment]
> > > please add "github.tarball_from"
> > 
> > 
> > @reneeotten How to use it with two subports correctly?
> 
> set it in both subports?

@reneeotten Done, also fixed ImageMagick enabling arg.

**Pull request merged**